### ISSUE TITLE
Add 'send email' button / checkbox in user creation flow

### ISSUE THREAD

**Issue by domdomegg** [raisenational/raise#251]
When creating users in the admin UI, we almost always end up emailing the user manually to let them know their account has been created.

Adding a button in the UI that when clicked automatically sent this email would be a nice to have.

Rough email template (maybe editable in a modal in the UI?):

```
To: <profile.email>
(maybe) Cc: <button_clicker.email>
From: <raise national email?>

Dear <profile.name>,

I've just updated your account on the Raise platform.

You should be able to login at https://www.joinraise.org/admin/ using your account <profile.email>, following these instructions: <Using the donations platform doc URL>.

<button_clicker.name>
```

(nb: maybe cut the using the donations platform doc URL after #248 done)
